eth/catalyst/api.go

```diff
@@ -103,9 +173,100 @@ func (api *consensusAPI) makeEnv(parent *types.Block, header *types.Header) (*bl
 	return env, nil
 }
 
+func (api *ConsensusAPI) PreparePayload(params AssembleBlockParams) (*PayloadResponse, error) {
+	data, err := api.assembleBlock(params)
```

Comment:
Just want to say this. The generated data is basically block. Accumulate the blocks non-stop is very bad.
Do we have any guarantee that when can we clean some cached blocks from the consensus layer?

Reply:
Previously I enabled the "transition" in this catalyst package by default, so that all the block creation via catalyst API are using the new rules. 

But now we pass the merger into the consensus engine, `engine = beacon.New(b.InnerEngine(), eth.Merger())`. So that it can happen:

for the first block creation, the TTDReached is still false, block is created with old rule.